src/libmeasurement_kit/dns/system_resolver_impl.hpp

```diff
@@ -60,7 +60,6 @@ void resolve_async_impl(ResolverContext *context) {
     int error = getaddrinfo(ctx->name.c_str(), nullptr, &ctx->hints, &sip);
     ctx->logger->log(MK_LOG_DEBUG2, "getaddrinfo result: %d", error);
     if (error) {
-        ctx->logger->warn("getaddrinfo failed: %s", gai_strerror(error));
```

Comment:
This is the only non-mechanic part of the merge: the file was moved and I applied this change manually. Contrary to other cases, this time the merge was done using `git merge -X ours stable`.